relbench/datasets/amazon.py

```diff
@@ -252,3 +252,29 @@ def fix_column(value):
                 ),
             }
         )
+
+    @property
+    def col_to_stype_dict(self) -> dict[str, dict[str, stype]]:
+        return {
+            "product": {
+                "product_id": stype.numerical,
+                "brand": stype.text_embedded,
+                "title": stype.text_embedded,
+                "description": stype.text_embedded,
+                "price": stype.numerical,
+                "category": stype.multicategorical,
+            },
+            "customer": {
+                "customer_id": stype.numerical,
+                "customer_name": stype.text_embedded,
+            },
+            "review": {
+                "review_text": stype.text_embedded,
+                "summary": stype.text_embedded,
+                "review_time": stype.timestamp,
+                "rating": stype.numerical,
+                "verified": stype.categorical,
+                "customer_id": stype.numerical,
+                "product_id": stype.numerical,
+            },
+        }
```

Comment:
This should not an inherent property of a dataset. Rather, it's something related to example code that uses pytorch frame.

Reply:
Moved back